src/app/[locale]/(pages)/accedi/__test__/page.test.tsx

```diff
@@ -1,25 +1,13 @@
-import { render, screen } from '@testing-library/react';
-import { NextIntlProvider } from 'next-intl';
+/* eslint-disable functional/immutable-data */
```

Comment:
Is this needed?

Reply:
Since we were getting errors while creating the tests with next-intl, we created a utils according to docs, have a look at -> '/app/[locale]/_utils/test-utils'

Reply:
Sorry, I mean the eslint disable directive. Is it needed? I see no error if I remove it.

Reply:
Ah, okay, you're right, I'll remove it